TensorFlow Lite build with CMake does not produce a binary on Windows

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Build/Install

### Source

source

### Tensorflow Version

tf 2.8

### Custom Code

No

### OS Platform and Distribution

Windows 10

### Mobile device

-

### Python version

-

### Bazel version

-

### GCC/Compiler version

MSVC 19.31.31104.0

### CUDA/cuDNN version

-

### GPU model and memory

-

### Current Behaviour?

```shell
I am building TF Lite with CMake. On Ubuntu and MacOS there is no problem and it works just fine.
However, on Windows the build just stops at `generating code...`.

I am following [this guide](https://www.tensorflow.org/lite/guide/build_cmake)
```


### Standalone code to reproduce the issue

```shell
bash
git clone https://github.com/tensorflow/tensorflow tensorflow_src
cd tensorflow_src
git checkout r2.8
cd ..

mkdir tflite_build
cd tflite_build
cmake ../tensorflow_src/tensorflow/lite/c
cmake --build .
```

When now checking the build directory with `ls -a tflite_build/` there is no output file.
```


### Relevant log output

```shell
PS C:\Users\Dario\dev\tflite_build> cmake ../tensorflow_src/tensorflow/lite/c
-- Building for: Visual Studio 17 2022
-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19044.
-- The C compiler identification is MSVC 19.31.31104.0
-- The CXX compiler identification is MSVC 19.31.31104.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Setting build type to Release, for debug builds use'-DCMAKE_BUILD_TYPE=Debug'.
-- Looking for pthread.h
-- Looking for pthread.h - not found
-- Found Threads: TRUE
-- Performing Test standard_math_library_linked_to_automatically
-- Performing Test standard_math_library_linked_to_automatically - Success
-- Standard libraries to link to explicitly: none
-- Performing Test COMPILER_SUPPORT_OPENMP
-- Performing Test COMPILER_SUPPORT_OPENMP - Success
-- Looking for a Fortran compiler
-- Looking for a Fortran compiler - NOTFOUND
-- Could NOT find CLANG_FORMAT: Found unsuitable version "0.0", but required is exact version "9" (found CLANG_FORMAT_EXECUTABLE-NOTFOUND)
--
-- Configured Eigen 3.4.90
--
-- Performing Test FARMHASH_HAS_BUILTIN_EXPECT
-- Performing Test FARMHASH_HAS_BUILTIN_EXPECT - Failed
-- Looking for _strtof_l
-- Looking for _strtof_l - found
-- Looking for _strtoui64_l
-- Looking for _strtoui64_l - found
-- The ASM compiler identification is MSVC
-- Found assembler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe
-- Downloading FP16 to C:/Users/Dario/dev/tflite_build/FP16-source (define FP16_SOURCE_DIR to avoid it)
-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19044.
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/Dario/dev/tflite_build/FP16-download
Microsoft (R) Build Engine version 17.1.0+ae57d105c for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  Checking Build System
  Creating directories for 'fp16'
  Building Custom Rule C:/Users/Dario/dev/tflite_build/FP16-download/CMakeLists.txt
  Performing download step (download, verify and extract) for 'fp16'
  -- Downloading...
     dst='C:/Users/Dario/dev/tflite_build/FP16-download/fp16-prefix/src/0a92994d729ff76a58f692d3028ca1b64b145d91.zip'
     timeout='none'
     inactivity timeout='none'
  -- Using src='https://github.com/Maratyszcza/FP16/archive/0a92994d729ff76a58f692d3028ca1b64b145d91.zip'
  -- verifying file...
         file='C:/Users/Dario/dev/tflite_build/FP16-download/fp16-prefix/src/0a92994d729ff76a58f692d3028ca1b64b145d91.zip'
  -- Downloading... done
  -- extracting...
       src='C:/Users/Dario/dev/tflite_build/FP16-download/fp16-prefix/src/0a92994d729ff76a58f692d3028ca1b64b145d91.zip'
       dst='C:/Users/Dario/dev/tflite_build/FP16-source'
  -- extracting... [tar xfz]
  -- extracting... [analysis]
  -- extracting... [rename]
  -- extracting... [clean up]
  -- extracting... done
  No update step for 'fp16'
  No patch step for 'fp16'
  No configure step for 'fp16'
  No build step for 'fp16'
  No install step for 'fp16'
  No test step for 'fp16'
  Completed 'fp16'
  Building Custom Rule C:/Users/Dario/dev/tflite_build/FP16-download/CMakeLists.txt
-- Downloading FXdiv to C:/Users/Dario/dev/tflite_build/FXdiv-source (define FXDIV_SOURCE_DIR to avoid it)
-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19044.
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/Dario/dev/tflite_build/FXdiv-download
Microsoft (R) Build Engine version 17.1.0+ae57d105c for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  Checking Build System
  Creating directories for 'fxdiv'
  Building Custom Rule C:/Users/Dario/dev/tflite_build/FXdiv-download/CMakeLists.txt
  Performing download step (download, verify and extract) for 'fxdiv'
  -- Downloading...
     dst='C:/Users/Dario/dev/tflite_build/FXdiv-download/fxdiv-prefix/src/b408327ac2a15ec3e43352421954f5b1967701d1.zip'
     timeout='none'
     inactivity timeout='none'
  -- Using src='https://github.com/Maratyszcza/FXdiv/archive/b408327ac2a15ec3e43352421954f5b1967701d1.zip'
  -- verifying file...
         file='C:/Users/Dario/dev/tflite_build/FXdiv-download/fxdiv-prefix/src/b408327ac2a15ec3e43352421954f5b1967701d1.zip'
  -- Downloading... done
  -- extracting...
       src='C:/Users/Dario/dev/tflite_build/FXdiv-download/fxdiv-prefix/src/b408327ac2a15ec3e43352421954f5b1967701d1.zip'
       dst='C:/Users/Dario/dev/tflite_build/FXdiv-source'
  -- extracting... [tar xfz]
  -- extracting... [analysis]
  -- extracting... [rename]
  -- extracting... [clean up]
  -- extracting... done
  No update step for 'fxdiv'
  No patch step for 'fxdiv'
  No configure step for 'fxdiv'
  No build step for 'fxdiv'
  No install step for 'fxdiv'
  No test step for 'fxdiv'
  Completed 'fxdiv'
  Building Custom Rule C:/Users/Dario/dev/tflite_build/FXdiv-download/CMakeLists.txt
-- Downloading pthreadpool to C:/Users/Dario/dev/tflite_build/pthreadpool-source (define PTHREADPOOL_SOURCE_DIR to avoid it)
-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19044.
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/Dario/dev/tflite_build/pthreadpool-download
Microsoft (R) Build Engine version 17.1.0+ae57d105c for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  Checking Build System
  Creating directories for 'pthreadpool'
  Building Custom Rule C:/Users/Dario/dev/tflite_build/pthreadpool-download/CMakeLists.txt
  Performing download step (download, verify and extract) for 'pthreadpool'
  -- Downloading...
     dst='C:/Users/Dario/dev/tflite_build/pthreadpool-download/pthreadpool-prefix/src/545ebe9f225aec6dca49109516fac02e973a3de2.zip'
     timeout='none'
     inactivity timeout='none'
  -- Using src='https://github.com/Maratyszcza/pthreadpool/archive/545ebe9f225aec6dca49109516fac02e973a3de2.zip'
  -- verifying file...
         file='C:/Users/Dario/dev/tflite_build/pthreadpool-download/pthreadpool-prefix/src/545ebe9f225aec6dca49109516fac02e973a3de2.zip'
  -- Downloading... done
  -- extracting...
       src='C:/Users/Dario/dev/tflite_build/pthreadpool-download/pthreadpool-prefix/src/545ebe9f225aec6dca49109516fac02e973a3de2.zip'
       dst='C:/Users/Dario/dev/tflite_build/pthreadpool-source'
  -- extracting... [tar xfz]
  -- extracting... [analysis]
  -- extracting... [rename]
  -- extracting... [clean up]
  -- extracting... done
  No update step for 'pthreadpool'
  No patch step for 'pthreadpool'
  No configure step for 'pthreadpool'
  No build step for 'pthreadpool'
  No install step for 'pthreadpool'
  No test step for 'pthreadpool'
  Completed 'pthreadpool'
  Building Custom Rule C:/Users/Dario/dev/tflite_build/pthreadpool-download/CMakeLists.txt
-- Downloading PSimd to C:/Users/Dario/dev/tflite_build/psimd-source (define PSIMD_SOURCE_DIR to avoid it)
-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19044.
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/Dario/dev/tflite_build/psimd-download
Microsoft (R) Build Engine version 17.1.0+ae57d105c for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  Checking Build System
  Creating directories for 'psimd'
  Building Custom Rule C:/Users/Dario/dev/tflite_build/psimd-download/CMakeLists.txt
  Performing download step (git clone) for 'psimd'
  Cloning into 'psimd-source'...
  Your branch is up to date with 'origin/master'.
  Already on 'master'
  Performing update step for 'psimd'
  HEAD is now at 072586a Fix psimd_qfma_f32 for FMA-enabled x86 processors
  No patch step for 'psimd'
  No configure step for 'psimd'
  No build step for 'psimd'
  No install step for 'psimd'
  No test step for 'psimd'
  Completed 'psimd'
  Building Custom Rule C:/Users/Dario/dev/tflite_build/psimd-download/CMakeLists.txt
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/Dario/dev/tflite_build
PS C:\Users\Dario\dev\tflite_build> cmake --build . -j
Microsoft (R) Build Engine version 17.1.0+ae57d105c for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  Checking Build System
  Building Custom Rule C:/Users/Dario/dev/tflite_build/clog/deps/clog/CMakeLists.txt
  clog.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/base/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/time/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/base/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/numeric/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/pthreadpool-source/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/base/CMakeLists.txt
  portable-api.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/flags/CMakeLists.txt
  log_severity.cc
  civil_time_detail.cc
  int128.cc
  spinlock_wait.cc
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/time/CMakeLists.txt
C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um\winbase.h(9531,5): warning C5105: macro expansion producing 'defined' has undefined behavior [C:\Users\Dario\dev\tflite_build\pthreadpool\pthreadpoo
l.vcxproj]
  commandlineflag.cc
  exponential_biased.cc
  time_zone_fixed.cc
  clog.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\clog-build\Debug\clog.lib
  absl_spinlock_wait.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\base\Debug\absl_spinlock_wait.lib
  absl_flags_commandlineflag_internal.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\flags\Debug\absl_flags_commandlineflag_internal.lib
  Building Custom Rule C:/Users/Dario/dev/tensorflow_src/tensorflow/lite/tools/cmake/modules/fft2d/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tensorflow_src/tensorflow/lite/tools/cmake/modules/farmhash/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/flatbuffers/CMakeLists.txt
  memory.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  absl_civil_time.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\time\Debug\absl_civil_time.lib
  fftsg.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/profiler/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  apply_multiplier.cc
  absl_exponential_biased.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\base\Debug\absl_exponential_biased.lib
  farmhash.cc
  idl_parser.cpp
  absl_log_severity.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\base\Debug\absl_log_severity.lib
  fft2d_fftsg.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\fft2d-build\Debug\fft2d_fftsg.lib
  instrumentation.cc
  time_zone_format.cc
  absl_int128.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\numeric\Debug\absl_int128.lib
C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um\winbase.h(9531,5): warning C5105: macro expansion producing 'defined' has undefined behavior [C:\Users\Dario\dev\tflite_build\pthreadpool\pthreadpoo
l.vcxproj]
  system_aligned_alloc.cc
C:\Users\Dario\dev\tflite_build\farmhash\src\farmhash.cc(394,1): warning C4319: '~': zero extending 'uint32_t' to 'T' of greater size [C:\Users\Dario\dev\tflite_build\_deps\farmhash-build\farmhash.vcxproj]
          with
          [
              T=uint64_t
          ]
C:\Users\Dario\dev\tflite_build\farmhash\src\farmhash.cc(404): message : see reference to function template instantiation 'T util::DebugTweak<uint64_t>(T)' being compiled [C:\Users\Dario\dev\tflite_build\_deps\f
armhash-build\farmhash.vcxproj]
          with
          [
              T=uint64_t
          ]
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  ruy_profiler_instrumentation.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\profiler\Debug\ruy_profiler_instrumentation.lib
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  have_built_path_for_avx.cc
  ruy_system_aligned_alloc.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_system_aligned_alloc.lib
  wait.cc
  farmhash.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\farmhash-build\Debug\farmhash.lib
  denormal.cc
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  have_built_path_for_avx512.cc
  have_built_path_for_avx2_fma.cc
  ruy_have_built_path_for_avx.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_have_built_path_for_avx.lib
  ruy_apply_multiplier.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_apply_multiplier.lib
  allocator.cc
  windows.c
  time_zone_if.cc
  ruy_denormal.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_denormal.lib
  ruy_have_built_path_for_avx2_fma.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_have_built_path_for_avx2_fma.lib
  ruy_have_built_path_for_avx512.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_have_built_path_for_avx512.lib
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/base/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tensorflow_src/tensorflow/lite/tools/cmake/modules/fft2d/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/cpuinfo/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um\winbase.h(9531,5): warning C5105: macro expansion producing 'defined' has undefined behavior [C:\Users\Dario\dev\tflite_build\pthreadpool\pthreadpoo
l.vcxproj]
  raw_logging.cc
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  init.c
  ruy_wait.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_wait.lib
  fftsg2d.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  ruy_allocator.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_allocator.lib
  time_zone_impl.cc
  block_map.cc
  prepacked_cache.cc
  fft2d_fftsg2d.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\fft2d-build\Debug\fft2d_fftsg2d.lib
  blocking_counter.cc
  api.c
  fastpath.c
  absl_raw_logging_internal.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\base\Debug\absl_raw_logging_internal.lib
  ruy_block_map.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_block_map.lib
  cache.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/debugging/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/types/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/base/CMakeLists.txt
  ruy_prepacked_cache.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_prepacked_cache.lib
C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um\winbase.h(9531,5): warning C5105: macro expansion producing 'defined' has undefined behavior [C:\Users\Dario\dev\tflite_build\pthreadpool\pthreadpoo
l.vcxproj]
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/types/CMakeLists.txt
  address_is_readable.cc
  ruy_blocking_counter.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_blocking_counter.lib
  time_zone_info.cc
  bad_optional_access.cc
  throw_delegate.cc
  elf_mem_image.cc
  idl_gen_text.cpp
  vdso_support.cc
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  info.c
  vendor.c
  Generating Code...
  uarch.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/base/CMakeLists.txt
  bad_variant_access.cc
  name.c
  topology.c
  isa.c
  descriptor.c
  deterministic.c
  absl_debugging_internal.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\debugging\Debug\absl_debugging_internal.lib
  Generating Code...
  cycleclock.cc
  Generating Code...
  thread_pool.cc
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/debugging/CMakeLists.txt
  init.c
  time_zone_libc.cc
  spinlock.cc
  absl_throw_delegate.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\base\Debug\absl_throw_delegate.lib
  absl_bad_optional_access.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\types\Debug\absl_bad_optional_access.lib
  pthreadpool.vcxproj -> C:\Users\Dario\dev\tflite_build\pthreadpool\Debug\pthreadpool.lib
  stacktrace.cc
  absl_bad_variant_access.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\types\Debug\absl_bad_variant_access.lib
  init.c
  init.c
  ruy_thread_pool.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_thread_pool.lib
  time_zone_lookup.cc
  sysinfo.cc
  absl_stacktrace.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\debugging\Debug\absl_stacktrace.lib
  reflection.cpp
  cpuinfo.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\cpuinfo-build\Debug\cpuinfo.lib
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/xnnpack/CMakeLists.txt
  cpuinfo.cc
  time_zone_posix.cc
  ruy_cpuinfo.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_cpuinfo.lib
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  zone_info_source.cc
  argmax-pooling-nhwc.c
  tune.cc
C:\Users\Dario\dev\tflite_build\xnnpack\src\operators\argmax-pooling-nhwc.c(234,81): warning C4090: 'function': different 'const' qualifiers [C:\Users\Dario\dev\tflite_build\_deps\xnnpack-build\XNNPACK.vcxproj]
  average-pooling-nhwc.c
  thread_identity.cc
C:\Users\Dario\dev\tflite_build\xnnpack\src\operators\average-pooling-nhwc.c(523,84): warning C4090: 'function': different 'const' qualifiers [C:\Users\Dario\dev\tflite_build\_deps\xnnpack-build\XNNPACK.vcxproj]
  binary-elementwise-nd.c
  channel-shuffle-nc.c
  Generating Code...
  constant-pad-nd.c
  convolution-nchw.c
  convolution-nhwc.c
  util.cpp
  ruy_tune.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_tune.lib
C:\Users\Dario\dev\tflite_build\xnnpack\src\operators\convolution-nhwc.c(1323,82): warning C4090: 'function': different 'const' qualifiers [C:\Users\Dario\dev\tflite_build\_deps\xnnpack-build\XNNPACK.vcxproj]
  deconvolution-nhwc.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
C:\Users\Dario\dev\tflite_build\xnnpack\src\operators\deconvolution-nhwc.c(576,112): warning C4090: 'function': different 'const' qualifiers [C:\Users\Dario\dev\tflite_build\_deps\xnnpack-build\XNNPACK.vcxproj]
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
C:\Users\Dario\dev\tflite_build\xnnpack\src\operators\deconvolution-nhwc.c(732,82): warning C4090: 'function': different 'const' qualifiers [C:\Users\Dario\dev\tflite_build\_deps\xnnpack-build\XNNPACK.vcxproj]
  depth-to-space-nchw2nhwc.c
  depth-to-space-nhwc.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  fully-connected-nc.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  absl_time_zone.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\time\Debug\absl_time_zone.lib
  global-average-pooling-ncw.c
  unscaledcycleclock.cc
  kernel_avx512.cc
  kernel_arm32.cc
  pack_avx.cc
  kernel_avx.cc
  global-average-pooling-nwc.c
  pack_avx2_fma.cc
  pack_avx512.cc
  Generating Code...
  kernel_avx2_fma.cc
  lut-elementwise-nc.c
  max-pooling-nhwc.c
C:\Users\Dario\dev\tflite_build\xnnpack\src\operators\max-pooling-nhwc.c(276,78): warning C4090: 'function': different 'const' qualifiers [C:\Users\Dario\dev\tflite_build\_deps\xnnpack-build\XNNPACK.vcxproj]
  prelu-nc.c
C:\Users\Dario\dev\tflite_build\ruy\ruy\kernel_avx512.cc(890,9): warning C4068: unknown pragma 'unroll' [C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\ruy_kernel_avx512.vcxproj]
  kernel_arm64.cc
  resize-bilinear-nchw.c
C:\Users\Dario\dev\tflite_build\xnnpack\src\operators\resize-bilinear-nchw.c(154,105): warning C4090: 'function': different 'const' qualifiers [C:\Users\Dario\dev\tflite_build\_deps\xnnpack-build\XNNPACK.vcxproj
]
  resize-bilinear-nhwc.c
  pack_arm.cc
C:\Users\Dario\dev\tflite_build\xnnpack\src\operators\resize-bilinear-nhwc.c(208,105): warning C4090: 'function': different 'const' qualifiers [C:\Users\Dario\dev\tflite_build\_deps\xnnpack-build\XNNPACK.vcxproj
]
  Generating Code...
  softmax-nc.c
  absl_base.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\base\Debug\absl_base.lib
  unary-elementwise-nc.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  Generating Code...
  ruy_pack_avx.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_pack_avx.lib
  ruy_pack_avx2_fma.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_pack_avx2_fma.lib
  ruy_kernel_avx2_fma.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_kernel_avx2_fma.lib
  Generating Code...
  ruy_kernel_avx512.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_kernel_avx512.lib
  Compiling...
  unpooling-nhwc.c
  ruy_kernel_avx.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_kernel_avx.lib
C:\Users\Dario\dev\tflite_build\xnnpack\src\operators\unpooling-nhwc.c(196,94): warning C4090: 'function': different 'const' qualifiers [C:\Users\Dario\dev\tflite_build\_deps\xnnpack-build\XNNPACK.vcxproj]
  abs.c
  ruy_pack_avx512.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_pack_avx512.lib
  add2.c
  ruy_kernel_arm.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_kernel_arm.lib
  argmax-pooling-2d.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/debugging/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/strings/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/base/CMakeLists.txt
  average-pooling-2d.c
  ctx.cc
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/hash/CMakeLists.txt
  bankers-rounding.c
  ceiling.c
  clamp.c
  low_level_alloc.cc
  ruy_pack_arm.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_pack_arm.lib
  ostringstream.cc
  demangle.cc
  convert.c
  convolution-2d.c
  city.cc
  deconvolution-2d.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/hash/CMakeLists.txt
  depth-to-space.c
  depthwise-convolution-2d.c
  divide.c
  elu.c
  absl_demangle_internal.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\debugging\Debug\absl_demangle_internal.lib
  flatbuffers.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\flatbuffers-build\Debug\flatbuffers.lib
  floor.c
  fully-connected.c
  wyhash.cc
  global-average-pooling-2d.c
  hardswish.c
  leaky-relu.c
  Generating Code...
  absl_city.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\hash\Debug\absl_city.lib
  utf8.cc
  escaping.cc
  Compiling...
  max-pooling-2d.c
  maximum2.c
  ruy_ctx.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_ctx.lib
  minimum2.c
  multiply2.c
  absl_wyhash.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\hash\Debug\absl_wyhash.lib
  negate.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  prelu.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  sigmoid.c
  softmax.c
  trmul.cc
  square-root.c
  context.cc
  prepare_packed_matrices.cc
  square.c
  absl_malloc_internal.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\base\Debug\absl_malloc_internal.lib
  squared-difference.c
  static-constant-pad.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/synchronization/CMakeLists.txt
  static-reshape.c
  static-resize-bilinear-2d.c
  graphcycles.cc
  subtract.c
  Generating Code...
  unpooling-2d.c
  datatype-strings.c
  operator-strings.c
  subgraph-strings.c
  absl_strings_internal.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\strings\Debug\absl_strings_internal.lib
  allocator.c
  Generating Code...
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/strings/CMakeLists.txt
  Compiling...
  init.c
  ascii.cc
  ruy_prepare_packed_matrices.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_prepare_packed_matrices.lib
  ruy_trmul.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_trmul.lib
  ruy_context.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_context.lib
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/ruy/ruy/CMakeLists.txt
  context_get_ctx.cc
  frontend.cc
  charconv.cc
  absl_graphcycles_internal.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\synchronization\Debug\absl_graphcycles_internal.lib
  memory-planner.c
  operator-delete.c
C:\Users\Dario\dev\tflite_build\xnnpack\src\operator-delete.c(29,44): warning C4090: 'function': different 'const' qualifiers [C:\Users\Dario\dev\tflite_build\_deps\xnnpack-build\XNNPACK.vcxproj]
  runtime.c
  subgraph.c
  tensor.c
  indirection.c
  operator-run.c
  ruy_frontend.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_frontend.lib
  packing.c
  exp2-k-over-64.c
  exp2-k-over-2048.c
  exp2minus-k-over-4.c
  exp2minus-k-over-8.c
  exp2minus-k-over-16.c
  exp2minus-k-over-64.c
  exp2minus-k-over-2048.c
  vcvt-scalar-float-x1.c
  ruy_context_get_ctx.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\ruy-build\ruy\Debug\ruy_context_get_ctx.lib
  vcvt-scalar-float-x4.c
  4x-scalar-c1.c
  escaping.cc
  9p8x-scalar-c1.c
  Generating Code...
  Compiling...
  9x-scalar-c1.c
  3x3s2p0p1c3x4-scalar-1x1.c
  up1x3-minmax-scalar-acc2.c
  up1x3-scalar-acc2.c
  up1x4-minmax-scalar-acc2.c
  up1x4-scalar-acc2.c
  up1x9-minmax-scalar-acc2.c
  up1x9-scalar-acc2.c
  up1x25-minmax-scalar-acc2.c
  up1x25-scalar-acc2.c
  3x3p1-minmax-scalar-2x1-acc2.c
  3x3p1-minmax-scalar-4x1.c
  3x3s2p1-minmax-scalar-1x1-acc2.c
  3x3s2p1-minmax-scalar-2x1-acc2.c
  5x5p2-minmax-scalar-1x1-acc5.c
  5x5p2-minmax-scalar-2x1-acc2.c
  5x5s2p2-minmax-scalar-1x1-acc5.c
  5x5s2p2-minmax-scalar-2x1-acc2.c
  vcvt-scalar-bitcast-x4.c
  charconv_bigint.cc
  vcvt-scalar-fabsf-x2.c
  Generating Code...
  Compiling...
  scalar-x1.c
  scalar-p4.c
  scalar-c2.c
  scalar-2x4.c
  scalar-p5-x4-acc2.c
  8x1-minmax-scalar.c
  8x2-minmax-scalar.c
  8x4-minmax-scalar.c
  vadd-minmax-scalar-x8.c
  vaddc-minmax-scalar-x8.c
  vdiv-minmax-scalar-x2.c
  vdiv-minmax-scalar-x8.c
  vdivc-minmax-scalar-x2.c
  vdivc-minmax-scalar-x8.c
  vmax-scalar-x8.c
  vmaxc-scalar-x8.c
  vmin-scalar-x8.c
  charconv_parse.cc
  vminc-scalar-x8.c
  vmul-minmax-scalar-x8.c
  vmulc-minmax-scalar-x8.c
  Generating Code...
  Compiling...
  vrdivc-minmax-scalar-x2.c
  vrdivc-minmax-scalar-x8.c
  vrsubc-minmax-scalar-x8.c
  vsqrdiff-scalar-x8.c
  vsqrdiffc-scalar-x8.c
  vsub-minmax-scalar-x8.c
  vsubc-minmax-scalar-x8.c
  memutil.cc
  vclamp-scalar-x4.c
  velu-scalar-rr2-lut16-p3-x2.c
  velu-scalar-rr2-lut16-p3-x4.c
  vhswish-scalar-x4.c
  vlrelu-scalar-x4.c
  c1-minmax-scalar-2x.c
  vrelu-scalar-x8.c
  vrndd-scalar-libm-x1.c
  vrndd-scalar-libm-x4.c
  vrndne-scalar-libm-x1.c
  vrndne-scalar-libm-x4.c
  vrndu-scalar-libm-x1.c
  vrndu-scalar-libm-x4.c
  match.cc
  Generating Code...
  Compiling...
  vrndz-scalar-libm-x1.c
  vrndz-scalar-libm-x4.c
  vsigmoid-scalar-lut64-p2-div-x2.c
  scalar-sqrt-x1.c
  vabs-scalar-x4.c
  vneg-scalar-x4.c
  vsqr-scalar-x4.c
  params-init.c
  numbers.cc
  7p7x-minmax-scalar-c4.c
  7x-minmax-scalar-c4.c
  lut-scalar-x4.c
  memcpy.c
  scalar-x16.c
  3x3s2p1c3x4-sse-2x2.c
  up8x3-minmax-sse.c
  up8x4-minmax-sse.c
  up8x9-minmax-sse.c
  up8x25-minmax-sse.c
  3x3p1-minmax-sse-2x4-acc2.c
  3x3s2p1-minmax-sse-1x4-acc3.c
  Generating Code...
  Compiling...
  5x5p2-minmax-sse-4x4.c
  5x5s2p2-minmax-sse-2x4.c
  sse-x4.c
  7p7x-minmax-sse-c4.c
  7x-minmax-sse-c4.c
  str_cat.cc
  sse-p8.c
  sse-c8.c
  sse.c
  32x1-minmax-sse.c
  vadd-minmax-sse-x8.c
  vaddc-minmax-sse-x8.c
  vdiv-minmax-sse-x8.c
  vdivc-minmax-sse-x8.c
  vmax-sse-x8.c
  vmaxc-sse-x8.c
  vmin-sse-x8.c
  vminc-sse-x8.c
  vmul-minmax-sse-x8.c
  vmulc-minmax-sse-x8.c
  vrdivc-minmax-sse-x8.c
  str_replace.cc
  Generating Code...
  Compiling...
  vrsubc-minmax-sse-x8.c
  vsqrdiff-sse-x8.c
  vsqrdiffc-sse-x8.c
  vsub-minmax-sse-x8.c
  vsubc-minmax-sse-x8.c
  vclamp-sse-x8.c
  vhswish-sse-x8.c
  vlrelu-sse-x8.c
  c4-minmax-sse-2x.c
  sse-sqrt-x4.c
  vabs-sse-x8.c
  str_split.cc
  vneg-sse-x8.c
  vsqr-sse-x8.c
  x4-sse.c
  vcvt-sse2-int16-x32.c
  4x-sse2-c4.c
  9p8x-sse2-c4.c
  9x-sse2-c4.c
  vcvt-sse2-x16.c
  sse2-2x8.c
  Generating Code...
  Compiling...
  sse2-p5-x20-acc2.c
  velu-sse2-rr2-lut16-p3-x12.c
  vlrelu-sse2-x8.c
  vrndd-sse2-x8.c
  vrndne-sse2-x8.c
  vrndu-sse2-x8.c
  vrndz-sse2-x8.c
  vsigmoid-sse2-lut64-p2-div-x8.c
  up8x9-minmax-fp32-sse2-mul16-add16.c
  string_view.cc
  up8x25-minmax-fp32-sse2-mul16-add16.c
  7p7x-minmax-sse2-c8-acc2.c
  7x-minmax-sse2-c8-acc2.c
  9p8x-minmax-sse2-c8.c
  9x-minmax-sse2-c8.c
  7p7x-minmax-sse2-c8.c
  7x-minmax-sse2-c8.c
  3x3p1-minmax-ssse3-2x4-acc2.c
  7p7x-minmax-ssse3-c8-acc2.c
  7x-minmax-ssse3-c8-acc2.c
  vcvt-sse41-int16-x16.c
  Generating Code...
  Compiling...
  vcvt-sse41-x8.c
  sse41-2x8.c
  vcvt-sse41-x32.c
  substitute.cc
  vlrelu-sse41-x8.c
  vrndd-sse41-x8.c
  vrndne-sse41-x8.c
  vrndu-sse41-x8.c
  vrndz-sse41-x8.c
  vsigmoid-sse41-lut64-p2-div-x8.c
  up8x9-minmax-fp32-sse41-mul16-add16.c
  up8x25-minmax-fp32-sse41-mul16-add16.c
  7p7x-minmax-sse41-c8-acc2.c
  7x-minmax-sse41-c8-acc2.c
  9p8x-minmax-sse41-c16.c
  sse41-x64.c
  Generating Code...
  Generating Code...
  9p8x-minmax-scalar-c1.c
  absl_strings.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\strings\Debug\absl_strings.lib
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/strings/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/hash/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/strings/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/flags/CMakeLists.txt
  9x-minmax-scalar-c1.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/time/CMakeLists.txt
  arg.cc
  commandlineflag.cc
  cord.cc
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/debugging/CMakeLists.txt
  hash.cc
  civil_time.cc
  symbolize.cc
  3x3s2p1c3x4-scalar-1x1.c
  3x3s2p1c3x4-scalar-1x1.c
  absl_flags_commandlineflag.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\flags\Debug\absl_flags_commandlineflag.lib
  7p7x-minmax-scalar-c1.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/flags/CMakeLists.txt
  private_handle_accessor.cc
  absl_hash.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\hash\Debug\absl_hash.lib
  bind.cc
  7x-minmax-scalar-c1.c
  clock.cc
  1x4-minmax-scalar.c
  absl_symbolize.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\debugging\Debug\absl_symbolize.lib
  1x4-relu-scalar.c
  cord_internal.cc
  absl_flags_private_handle_accessor.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\flags\Debug\absl_flags_private_handle_accessor.lib
  1x4-scalar.c
  extension.cc
  duration.cc
  2x4-minmax-scalar.c
  2x4-relu-scalar.c
  cord_rep_ring.cc
  float_conversion.cc
  2x4-scalar.c
  4x2-minmax-scalar.c
  4x2-relu-scalar.c
  format.cc
  Generating Code...
  4x2-scalar.c
  output.cc
  4x4-minmax-scalar.c
  absl_cord.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\strings\Debug\absl_cord.lib
  4x4-relu-scalar.c
  time.cc
  parser.cc
  4x4-scalar.c
  1x4-minmax-scalar.c
  1x4-relu-scalar.c
  Generating Code...
  Generating Code...
  1x4-scalar.c
  absl_time.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\time\Debug\absl_time.lib
  2x4-minmax-scalar.c
  absl_str_format_internal.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\strings\Debug\absl_str_format_internal.lib
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/flags/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/status/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/synchronization/CMakeLists.txt
  2x4-relu-scalar.c
  marshalling.cc
  status.cc
  barrier.cc
  2x4-scalar.c
  4x2-minmax-scalar.c
  4x2-relu-scalar.c
  blocking_counter.cc
  4x2-scalar.c
  absl_flags_marshalling.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\flags\Debug\absl_flags_marshalling.lib
  status_payload_printer.cc
  4x4-minmax-scalar.c
  create_thread_identity.cc
  4x4-relu-scalar.c
  4x4-scalar.c
  Generating Code...
  9p8x-minmax-scalar-c1.c
  per_thread_sem.cc
  absl_status.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\status\Debug\absl_status.lib
  9p8x-minmax-scalar-c1.c
  9x-minmax-scalar-c1.c
  vcvt-scalar-magic-iminmax-x1.c
  waiter.cc
  vcvt-scalar-magic-iminmax-x4.c
  vcvt-scalar-magic-iminmax-x1.c
  vcvt-scalar-magic-iminmax-x4.c
  notification.cc
  scalar.c
  up2x9-minmax-fp32-scalar-magic.c
  up2x25-minmax-fp32-scalar-magic.c
  mutex.cc
  1x2-minmax-fp32-scalar-magic.c
  1x4-minmax-fp32-scalar-magic.c
  2x2-minmax-fp32-scalar-magic.c
  Generating Code...
  4x4-minmax-fp32-scalar-magic.c
  absl_synchronization.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\synchronization\Debug\absl_synchronization.lib
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/flags/CMakeLists.txt
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/container/CMakeLists.txt
  1x2-minmax-fp32-scalar-magic.c
  program_name.cc
  hashtablez_sampler.cc
  1x4-minmax-fp32-scalar-magic.c
  2x2-minmax-fp32-scalar-magic.c
  hashtablez_sampler_force_weak_definition.cc
  absl_flags_program_name.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\flags\Debug\absl_flags_program_name.lib
  4x4-minmax-fp32-scalar-magic.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/flags/CMakeLists.txt
  usage_config.cc
  up1x9-minmax-fp32-scalar-magic.c
  Generating Code...
  up1x25-minmax-fp32-scalar-magic.c
  absl_hashtablez_sampler.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\container\Debug\absl_hashtablez_sampler.lib
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/container/CMakeLists.txt
  raw_hash_set.cc
  up2x9-minmax-fp32-scalar-magic.c
  absl_flags_config.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\flags\Debug\absl_flags_config.lib
  up2x25-minmax-fp32-scalar-magic.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/flags/CMakeLists.txt
  flag.cc
  vcvt-scalar-x1.c
  vcvt-scalar-x4.c
  absl_raw_hash_set.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\container\Debug\absl_raw_hash_set.lib
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/flags/CMakeLists.txt
  7p7x-minmax-scalar-c1.c
  reflection.cc
  7x-minmax-scalar-c1.c
  absl_flags_internal.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\flags\Debug\absl_flags_internal.lib
  1x2-minmax-fp32-scalar-magic.c
  1x4-minmax-fp32-scalar-magic.c
  1x4-minmax-rndnu-scalar.c
  2x2-minmax-fp32-scalar-magic.c
  absl_flags_reflection.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\flags\Debug\absl_flags_reflection.lib
  3x4-minmax-rndnu-scalar.c
  Building Custom Rule C:/Users/Dario/dev/tflite_build/abseil-cpp/absl/flags/CMakeLists.txt
  flag.cc
  4x4-minmax-fp32-scalar-magic.c
  1x2-minmax-fp32-scalar-magic.c
  1x4-minmax-fp32-scalar-magic.c
  absl_flags.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\abseil-cpp-build\absl\flags\Debug\absl_flags.lib
  1x4-minmax-rndnu-scalar.c
  2x2-minmax-fp32-scalar-magic.c
  3x4-minmax-rndnu-scalar.c
  4x4-minmax-fp32-scalar-magic.c
  minmax-scalar-x1.c
  minmax-scalar-x4.c
  minmax-scalar-x1.c
  minmax-scalar-x4.c
  minmax-fp32-scalar-x4.c
  minmax-fp32-scalar-x4.c
  9p8x-minmax-scalar-c1.c
  9x-minmax-scalar-c1.c
  up1x9-minmax-fp32-scalar-magic.c
  up1x25-minmax-fp32-scalar-magic.c
  up2x9-minmax-fp32-scalar-magic.c
  up2x25-minmax-fp32-scalar-magic.c
  vcvt-scalar-x1.c
  vcvt-scalar-x4.c
  7p7x-minmax-scalar-c1.c
  7x-minmax-scalar-c1.c
  1x2-minmax-fp32-scalar-magic.c
  1x4-minmax-fp32-scalar-magic.c
  2x2-minmax-fp32-scalar-magic.c
  4x4-minmax-fp32-scalar-magic.c
  1x2-minmax-fp32-scalar-magic.c
  1x4-minmax-fp32-scalar-magic.c
  2x2-minmax-fp32-scalar-magic.c
  4x4-minmax-fp32-scalar-magic.c
  minmax-scalar-x1.c
  minmax-scalar-x4.c
  minmax-scalar-x1.c
  minmax-scalar-x4.c
  minmax-fp32-scalar-x4.c
  minmax-fp32-scalar-x4.c
  scalar-c1.c
  9p8x-minmax-scalar-c1.c
  scalar-x4.c
  scalar-c1.c
  scalar.c
  9p8x-minmax-scalar-c1.c
  scalar.c
  scalar-x4.c
  x2-scalar.c
  x3-scalar.c
  x4-scalar.c
  xm-scalar.c
  scalar.c
  x2-scalar.c
  x3-scalar.c
  x4-scalar.c
  scalar.c
  x2-scalar.c
  x3-scalar.c
  x4-scalar.c
  xm-scalar.c
  scalar.c
  9p8x-minmax-sse-c4.c
  9x-minmax-sse-c4.c
  1x8-minmax-sse-load1.c
  4x2c4-minmax-sse.c
  4x8-minmax-sse-load1.c
  1x8-minmax-sse-load1.c
  4x2c4-minmax-sse.c
  4x8-minmax-sse-load1.c
  9p8x-minmax-sse-c4.c
  9p8x-minmax-sse-c4.c
  9x-minmax-sse-c4.c
  vcvt-sse2-x32.c
  vcvt-sse2-x32.c
  up8x9-minmax-fp32-sse2-mul16.c
  up8x25-minmax-fp32-sse2-mul16.c
  1x4c8-minmax-fp32-sse2-ld64.c
  3x4c8-minmax-fp32-sse2-ld64.c
  1x4c8-minmax-fp32-sse2-ld64.c
  3x4c8-minmax-fp32-sse2-ld64.c
  vcvt-sse2-x32.c
  1x4c8-minmax-fp32-sse2-ld64.c
  3x4c8-minmax-fp32-sse2-ld64.c
  1x4c8-minmax-fp32-sse2-ld64.c
  3x4c8-minmax-fp32-sse2-ld64.c
  minmax-sse2-mul16-ld64-x8.c
  minmax-sse2-mul16-ld64-x8.c
  minmax-fp32-sse2-mul16-ld64-x8.c
  minmax-fp32-sse2-mul16-ld64-x8.c
  up8x9-minmax-fp32-sse2-mul16.c
  up8x25-minmax-fp32-sse2-mul16.c
  vcvt-sse2-x32.c
  1x4c8-minmax-fp32-sse2-ld64.c
  3x4c8-minmax-fp32-sse2-ld64.c
  1x4c8-minmax-fp32-sse2-ld64.c
  3x4c8-minmax-fp32-sse2-ld64.c
  minmax-sse2-mul16-ld64-x8.c
  minmax-sse2-mul16-ld64-x8.c
  minmax-fp32-sse2-mul16-ld64-x8.c
  minmax-fp32-sse2-mul16-ld64-x8.c
  sse2-c8.c
  9p8x-minmax-sse2-c16.c
  sse2-x64.c
  sse2-c8.c
  9p8x-minmax-sse2-c16.c
  sse2.c
  sse2-x64.c
  x2-sse2.c
  x3-sse2.c
  x4-sse2.c
  xm-sse2.c
  sse2.c
  x2-sse2.c
  x3-sse2.c
  x4-sse2.c
  xm-sse2.c
  sse2-x64.c
  sse2.c
  up8x9-minmax-fp32-sse41-mul16.c
  up8x25-minmax-fp32-sse41-mul16.c
  1x4c8-minmax-fp32-sse41-ld64.c
  3x4c8-minmax-fp32-sse41-ld64.c
  1x4c8-minmax-fp32-sse41-ld64.c
  3x4c8-minmax-fp32-sse41-ld64.c
  vcvt-sse41-x16.c
  1x4c8-minmax-fp32-sse41-ld64.c
  3x4c8-minmax-fp32-sse41-ld64.c
  1x4c8-minmax-fp32-sse41-ld64.c
  3x4c8-minmax-fp32-sse41-ld64.c
  minmax-sse41-mul16-ld64-x8.c
  minmax-sse41-mul16-ld64-x8.c
  minmax-fp32-sse41-mul16-ld64-x16.c
  minmax-fp32-sse41-mul16-ld64-x16.c
  up8x9-minmax-fp32-sse41-mul16.c
  up8x25-minmax-fp32-sse41-mul16.c
  vcvt-sse41-x16.c
  1x4c8-minmax-fp32-sse41-ld64.c
  3x4c8-minmax-fp32-sse41-ld64.c
  1x4c8-minmax-fp32-sse41-ld64.c
  3x4c8-minmax-fp32-sse41-ld64.c
  minmax-sse41-mul16-ld64-x8.c
  minmax-sse41-mul16-ld64-x8.c
  minmax-fp32-sse41-mul16-ld64-x16.c
  minmax-fp32-sse41-mul16-ld64-x16.c
  sse41-c16.c
  sse41-c16.c
  vcvt-avx-int16-x16.c
  up8x25-minmax-avx.c
  up16x3-minmax-avx.c
  up16x4-minmax-avx.c
  up16x9-minmax-avx.c
  vcvt-avx-x24.c
  avx-2x16.c
  vadd-minmax-avx-x16.c
  vaddc-minmax-avx-x16.c
  vdiv-minmax-avx-x16.c
  vdivc-minmax-avx-x16.c
  vmax-avx-x16.c
  vmaxc-avx-x16.c
  vmin-avx-x16.c
  vminc-avx-x16.c
  vmul-minmax-avx-x16.c
  vmulc-minmax-avx-x16.c
  vrdivc-minmax-avx-x16.c
  vrsubc-minmax-avx-x16.c
  vsqrdiff-avx-x16.c
  Generating Code...
  Compiling...
  vsqrdiffc-avx-x16.c
  vsub-minmax-avx-x16.c
  vsubc-minmax-avx-x16.c
  vclamp-avx-x16.c
  velu-avx-rr2-lut4-p4-perm-x32.c
  vhswish-avx-x16.c
  vlrelu-avx-x16.c
  vrndd-avx-x16.c
  vrndne-avx-x16.c
  vrndu-avx-x16.c
  vrndz-avx-x16.c
  vsigmoid-avx-rr2-p5-nr2-x40.c
  avx-sqrt-x8.c
  vabs-avx-x16.c
  vneg-avx-x16.c
  vsqr-avx-x16.c
  up16x9-minmax-fp32-avx-mul16.c
  up16x25-minmax-fp32-avx-mul16.c
  lut-avx-x64.c
  up16x9-minmax-fp32-xop-mul32.c
  Generating Code...
  Compiling...
  up16x25-minmax-fp32-xop-mul32.c
  up8x25-minmax-fma3.c
  up16x3-minmax-fma3.c
  up16x4-minmax-fma3.c
  up16x9-minmax-fma3.c
  vhswish-fma3-x16.c
  Generating Code...
  1x16-minmax-avx-broadcast.c
  5x16-minmax-avx-broadcast.c
  1x16-minmax-avx-broadcast.c
  5x16-minmax-avx-broadcast.c
  up16x9-minmax-fp32-avx-mul16-add16.c
  up16x25-minmax-fp32-avx-mul16-add16.c
  1x4c8-minmax-fp32-avx-ld128.c
  2x4c8-minmax-fp32-avx-ld128.c
  1x4c8-minmax-fp32-avx-ld128.c
  2x4c8-minmax-fp32-avx-ld128.c
  up16x9-minmax-fp32-avx-mul16-add16.c
  up16x25-minmax-fp32-avx-mul16-add16.c
  1x4c8-minmax-fp32-avx-ld128.c
  2x4c8-minmax-fp32-avx-ld128.c
  1x4c8-minmax-fp32-avx-ld128.c
  2x4c8-minmax-fp32-avx-ld128.c
  minmax-avx-mul32-ld32-x8.c
  minmax-avx-mul32-ld32-x8.c
  minmax-fp32-avx-mul16-ld64-x16.c
  minmax-fp32-avx-mul16-ld64-x16.c
  1x4c8-minmax-fp32-avx-ld128.c
  2x4c8-minmax-fp32-avx-ld128.c
  1x4c8-minmax-fp32-avx-ld128.c
  2x4c8-minmax-fp32-avx-ld128.c
  minmax-avx-mul32-ld32-x8.c
  minmax-avx-mul32-ld32-x8.c
  minmax-fp32-avx-mul16-ld64-x16.c
  minmax-fp32-avx-mul16-ld64-x16.c
  vcvt-f16c-x16.c
  vcvt-f16c-x16.c
C:\Users\Dario\dev\tflite_build\xnnpack\src\f32-f16-vcvt\gen\vcvt-f16c-x16.c(38,1): warning C4556: value of intrinsic immediate argument '8' is out of range '0 - 7' [C:\Users\Dario\dev\tflite_build\_deps\xnnpack
-build\XNNPACK.vcxproj]
C:\Users\Dario\dev\tflite_build\xnnpack\src\f32-f16-vcvt\gen\vcvt-f16c-x16.c(39,1): warning C4556: value of intrinsic immediate argument '8' is out of range '0 - 7' [C:\Users\Dario\dev\tflite_build\_deps\xnnpack
-build\XNNPACK.vcxproj]
C:\Users\Dario\dev\tflite_build\xnnpack\src\f32-f16-vcvt\gen\vcvt-f16c-x16.c(46,1): warning C4556: value of intrinsic immediate argument '8' is out of range '0 - 7' [C:\Users\Dario\dev\tflite_build\_deps\xnnpack
-build\XNNPACK.vcxproj]
C:\Users\Dario\dev\tflite_build\xnnpack\src\f32-f16-vcvt\gen\vcvt-f16c-x16.c(58,1): warning C4556: value of intrinsic immediate argument '8' is out of range '0 - 7' [C:\Users\Dario\dev\tflite_build\_deps\xnnpack
-build\XNNPACK.vcxproj]
C:\Users\Dario\dev\tflite_build\xnnpack\src\f32-f16-vcvt\gen\vcvt-f16c-x16.c(62,1): warning C4556: value of intrinsic immediate argument '8' is out of range '0 - 7' [C:\Users\Dario\dev\tflite_build\_deps\xnnpack
-build\XNNPACK.vcxproj]
  up16x9-minmax-fp32-xop-mul16-add16.c
  up16x25-minmax-fp32-xop-mul16-add16.c
  1x4c8-minmax-fp32-xop-ld64.c
  2x4c8-minmax-fp32-xop-ld64.c
  1x4c8-minmax-fp32-xop-ld64.c
  2x4c8-minmax-fp32-xop-ld64.c
  up16x9-minmax-fp32-xop-mul16-add16.c
  up16x25-minmax-fp32-xop-mul16-add16.c
  1x4c8-minmax-fp32-xop-ld64.c
  2x4c8-minmax-fp32-xop-ld64.c
  1x4c8-minmax-fp32-xop-ld64.c
  2x4c8-minmax-fp32-xop-ld64.c
  minmax-xop-mul32-ld32-x8.c
  minmax-xop-mul32-ld32-x8.c
  1x4c8-minmax-fp32-xop-ld64.c
  2x4c8-minmax-fp32-xop-ld64.c
  1x4c8-minmax-fp32-xop-ld64.c
  2x4c8-minmax-fp32-xop-ld64.c
  minmax-xop-mul32-ld32-x8.c
  minmax-xop-mul32-ld32-x8.c
  1x16-minmax-fma3-broadcast.c
  1x16s4-minmax-fma3-broadcast.c
  4x16s4-minmax-fma3-broadcast.c
  5x16-minmax-fma3-broadcast.c
  1x16-minmax-fma3-broadcast.c
  1x16s4-minmax-fma3-broadcast.c
  4x16s4-minmax-fma3-broadcast.c
  5x16-minmax-fma3-broadcast.c
  velu-avx2-rr1-lut4-p4-perm-x56.c
  vsigmoid-avx2-rr1-p5-div-x40.c
  lut-avx2-x128.c
  Generating Code...
  up16x9-minmax-fp32-avx2-mul32.c
  up16x25-minmax-fp32-avx2-mul32.c
  1x8c8-minmax-fp32-avx2.c
  3x8c8-minmax-fp32-avx2.c
  1x8c8-minmax-fp32-avx2.c
  3x8c8-minmax-fp32-avx2.c
  up16x9-minmax-fp32-avx2-mul32.c
  up16x25-minmax-fp32-avx2-mul32.c
  1x8c8-minmax-fp32-avx2.c
  3x8c8-minmax-fp32-avx2.c
  1x8c8-minmax-fp32-avx2.c
  3x8c8-minmax-fp32-avx2.c
  minmax-avx2-mul32-ld64-x16.c
  minmax-avx2-mul32-ld64-x16.c
  up16x9-minmax-fp32-avx2-mul32.c
  up16x25-minmax-fp32-avx2-mul32.c
  1x8c8-minmax-fp32-avx2.c
  3x8c8-minmax-fp32-avx2.c
  1x8c8-minmax-fp32-avx2.c
  3x8c8-minmax-fp32-avx2.c
  minmax-avx2-mul32-ld64-x16.c
  minmax-avx2-mul32-ld64-x16.c
  up16x3-minmax-avx512f.c
  up16x4-minmax-avx512f.c
  up16x9-minmax-avx512f.c
  up16x25-minmax-avx512f.c
  avx512f-2x16.c
  vadd-minmax-avx512f-x32.c
  vaddc-minmax-avx512f-x32.c
  vdiv-minmax-avx512f-x32.c
  vdivc-minmax-avx512f-x32.c
  vmax-avx512f-x32.c
  vmaxc-avx512f-x32.c
  vmin-avx512f-x32.c
  vminc-avx512f-x32.c
  vmul-minmax-avx512f-x32.c
  vmulc-minmax-avx512f-x32.c
  vrdivc-minmax-avx512f-x32.c
  vrsubc-minmax-avx512f-x32.c
  vsqrdiff-avx512f-x32.c
  vsqrdiffc-avx512f-x32.c
  vsub-minmax-avx512f-x32.c
  Generating Code...
  Compiling...
  vsubc-minmax-avx512f-x32.c
  vclamp-avx512f-x16.c
  velu-avx512f-rr1-lut16-p3-perm-x64.c
  vhswish-avx512f-x16.c
  vlrelu-avx512f-x16.c
  vrndd-avx512f-x16.c
  vrndne-avx512f-x16.c
  vrndu-avx512f-x16.c
  vrndz-avx512f-x16.c
  vsigmoid-avx512f-rr2-lut32-p2-perm2-scalef-div-x64.c
  vabs-avx512f-x16.c
  vneg-avx512f-x16.c
  vsqr-avx512f-x16.c
  lut-avx512skx-vpshufb-x64.c
  Generating Code...
  1x16-minmax-avx512f-broadcast.c
  7x16-minmax-avx512f-broadcast.c
  1x16-minmax-avx512f-broadcast.c
  7x16-minmax-avx512f-broadcast.c
  vcvt-avx512skx-x16.c
  vcvt-avx512skx-x16.c
  up32x9-minmax-fp32-avx512skx-mul32.c
  up32x25-minmax-fp32-avx512skx-mul32.c
  1x16c8-minmax-fp32-avx512skx.c
  4x16c8-minmax-fp32-avx512skx.c
  1x16c8-minmax-fp32-avx512skx.c
  4x16c8-minmax-fp32-avx512skx.c
  up32x9-minmax-fp32-avx512skx-mul32.c
  up32x25-minmax-fp32-avx512skx-mul32.c
  1x16c8-minmax-fp32-avx512skx.c
  4x16c8-minmax-fp32-avx512skx.c
  1x16c8-minmax-fp32-avx512skx.c
  4x16c8-minmax-fp32-avx512skx.c
  minmax-avx512skx-mul32-ld128-x16.c
  minmax-avx512skx-mul32-ld128-x16.c
  up32x9-minmax-fp32-avx512skx-mul32.c
  up32x25-minmax-fp32-avx512skx-mul32.c
  1x16c8-minmax-fp32-avx512skx.c
  4x16c8-minmax-fp32-avx512skx.c
  1x16c8-minmax-fp32-avx512skx.c
  4x16c8-minmax-fp32-avx512skx.c
  minmax-avx512skx-mul32-ld128-x16.c
  minmax-avx512skx-mul32-ld128-x16.c
  XNNPACK.vcxproj -> C:\Users\Dario\dev\tflite_build\_deps\xnnpack-build\Debug\XNNPACK.lib
  Building Custom Rule C:/Users/Dario/dev/tensorflow_src/tensorflow/lite/CMakeLists.txt
  error_reporter.cc
  flatbuffer_conversions.cc
  op_resolver.cc
  subgraph.cc
  c_api.cc
  c_api_experimental.cc
  c_api_for_testing.cc
  nnapi_delegate_disabled.cc
  interpreter_utils.cc
  serialization.cc
  telemetry.cc
  utils.cc
  xnnpack_delegate.cc
  initialization_status.cc
  resource_variable.cc
  static_hashtable.cc
  cpu_check.cc
  neon_tensor_utils.cc
  sse_tensor_utils.cc
  portable_tensor_utils.cc
  Generating Code...
  Compiling...
  kernel_utils.cc
  mfcc_dct.cc
  mfcc_mel_filterbank.cc
  spectrogram.cc
  transpose_utils.cc
  activations.cc
  add.cc
  add_n.cc
  arg_min_max.cc
  assign_variable.cc
  atan2.cc
  audio_spectrogram.cc
  basic_rnn.cc
  batch_matmul.cc
  batch_to_space_nd.cc
  bidirectional_sequence_lstm.cc
  bidirectional_sequence_rnn.cc
  broadcast_args.cc
  broadcast_to.cc
  bucketize.cc
  Generating Code...
  Compiling...
  call_once.cc
  cast.cc
  ceil.cc
  comparisons.cc
  complex_support.cc
  concatenation.cc
  conv.cc
  conv3d.cc
  conv3d_transpose.cc
  cpu_backend_context.cc
  cpu_backend_gemm_eigen.cc
  cumsum.cc
  densify.cc
  deprecated_backends.cc
  depth_to_space.cc
  depthwise_conv.cc
  dequantize.cc
  detection_postprocess.cc
  div.cc
  eigen_support.cc
  Generating Code...
  Compiling...
  elementwise.cc
  embedding_lookup.cc
  embedding_lookup_sparse.cc
  exp.cc
  expand_dims.cc
  fake_quant.cc
  fill.cc
  floor.cc
  floor_div.cc
  floor_mod.cc
  fully_connected.cc
  gather.cc
  gather_nd.cc
  gru_cell.cc
  hashtable.cc
  hashtable_find.cc
  hashtable_import.cc
  hashtable_lookup.cc
  hashtable_size.cc
  if.cc
  Generating Code...
  Compiling...
  irfft2d.cc
  kernel_util.cc
  l2norm.cc
  local_response_norm.cc
  logical.cc
  lsh_projection.cc
  lstm.cc
  lstm_eval.cc
  matrix_diag.cc
  matrix_set_diag.cc
  maximum_minimum.cc
  mirror_pad.cc
  mul.cc
  multinomial.cc
  neg.cc
  non_max_suppression.cc
  numeric_verify.cc
  one_hot.cc
  pack.cc
  pad.cc
  Generating Code...
  Compiling...
  pooling.cc
  pooling3d.cc
  pow.cc
  quantize.cc
  random_ops.cc
C:\Users\Dario\dev\tensorflow_src\tensorflow/core/lib/random/random_distributions_utils.h(78,27): error C2065: 'M_PI': undeclared identifier [C:\Users\Dario\dev\tflite_build\tensorflow-lite\tensorflow-lite.vcxpr
oj]
C:\Users\Dario\dev\tensorflow_src\tensorflow/core/lib/random/random_distributions_utils.h(78,15): error C2737: 'v1': const object must be initialized [C:\Users\Dario\dev\tflite_build\tensorflow-lite\tensorflow-l
ite.vcxproj]
  random_standard_normal_custom.cc
  random_uniform_custom.cc
  range.cc
  rank.cc
  read_variable.cc
  reduce.cc
  register.cc
  register_ref.cc
  reshape.cc
  resize_bilinear.cc
  resize_nearest_neighbor.cc
  reverse.cc
  reverse_sequence.cc
  rfft2d.cc
  roll.cc
  Generating Code...
  Compiling...
  round.cc
  scatter_nd.cc
  segment_sum.cc
  select.cc
  shape.cc
  sign.cc
  skip_gram.cc
  slice.cc
  space_to_batch_nd.cc
  space_to_depth.cc
  sparse_to_dense.cc
  split.cc
  split_v.cc
  squared_difference.cc
  squeeze.cc
  strided_slice.cc
  sub.cc
  svdf.cc
  table.cc
  tile.cc
  Generating Code...
  Compiling...
  topk_v2.cc
  transpose.cc
  transpose_conv.cc
  unidirectional_sequence_gru.cc
  unidirectional_sequence_lstm.cc
  unidirectional_sequence_rnn.cc
  unique.cc
  unpack.cc
  var_handle.cc
  where.cc
  while.cc
  zeros_like.cc
  nnapi_implementation_disabled.cc
  allocation.cc
  arena_planner.cc
  create_op_resolver_with_builtin_ops.cc
  external_cpu_backend_context.cc
  graph_info.cc
  interpreter.cc
  interpreter_builder.cc
  Generating Code...
  Compiling...
  interpreter_builder_experimental.cc
  interpreter_experimental.cc
  minimal_logging.cc
  minimal_logging_default.cc
C:\Users\Dario\dev\tensorflow_src\tensorflow\lite\minimal_logging_default.cc(28,9): warning C4068: unknown pragma 'clang' [C:\Users\Dario\dev\tflite_build\tensorflow-lite\tensorflow-lite.vcxproj]
C:\Users\Dario\dev\tensorflow_src\tensorflow\lite\minimal_logging_default.cc(29,9): warning C4068: unknown pragma 'clang' [C:\Users\Dario\dev\tflite_build\tensorflow-lite\tensorflow-lite.vcxproj]
C:\Users\Dario\dev\tensorflow_src\tensorflow\lite\minimal_logging_default.cc(31,9): warning C4068: unknown pragma 'clang' [C:\Users\Dario\dev\tflite_build\tensorflow-lite\tensorflow-lite.vcxproj]
  mmap_allocation_disabled.cc
  model_builder.cc
  mutable_op_resolver.cc
  optional_debug_tools.cc
  signature_runner.cc
  simple_memory_arena.cc
  simple_memory_arena_debug_dump.cc
  simple_planner.cc
  stderr_reporter.cc
  string_util.cc
  tflite_with_xnnpack_optional.cc
  util.cc
  platform_profiler.cc
  sparsity_format_converter.cc
  schema_utils.cc
  Generating Code...
```
</details>

--- a/tensorflow/lite/g3doc/guide/build_cmake.md
+++ b/tensorflow/lite/g3doc/guide/build_cmake.md
@@ -290,6 +290,8 @@ cmake --build . -j
 
 This command generates the following shared library in the current directory.
 
+**Note:** On Windows system, you can find the `tensorflowlite_c.dll` under `debug` directory.
+
 Platform | Library name
 -------- | ---------------------------
 Linux    | `libtensorflowlite_c.so`

